docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -1049,6 +1049,26 @@ Then local storage item with key `token` does not exist
 include::partial$proxy-steps.adoc[]
 
 === Window Steps
+==== Drag and drop steps
```

Comment:
it has actually nothing about windows, I would say it's more about mouse actions